src/lib/expansion/expansion-panel-header.html

```diff
@@ -0,0 +1,7 @@
+<span class="mat-content">
+  <ng-content select="[title]"></ng-content>
+  <ng-content select="[description]"></ng-content>
```

Comment:
This is for matching the material spec for the headers.

The first demo actually does use it, updated the demo to pack it a bit more obvious